modules/olm-injecting-custom-ca.adoc

```diff
@@ -68,3 +68,8 @@ spec:
 <4> `ca-bundle.crt` is required as the config map key.
 <5> `tls-ca-bundle.pem` is required as the config map path.
 <6> Create a `trusted-ca` volume mount.
++
+[NOTE]
+====
+If an Operator pod fails with the x509 certificate validation error, set the  `mountPath` as `/etc/ssl/certs`.
```

Comment:
Not all x509 errors can be fixed by this operation.

Reply:
@jianzhangbjz Can we add any additional information or can we rearrange the sentence to something like - 
`You can avoid some x509 certificate validation errors by setting the `mountPath` as `/etc/ssl/certs`. ?